packages/client/src/runtime/utils/common.ts

```diff
@@ -65,7 +65,7 @@ export const needNamespace = {
   Decimal: 'Decimal',
 }
 
-export function needsNamespace(fieldType: DMMF.Field['type'], dmmf: DMMFClass): boolean {
+export function needsNamespace(fieldType: DMMF.SchemaField['outputType']['type'], dmmf: DMMFHelper): boolean {
```

Comment:
What is that `SchemaField` change about?

Reply:
It's a change that I made last time after removing some legacy logic (after talking to Dom). I just realized that the type was wrong (by looking at its usage).